.travis.yml

```diff
@@ -3,8 +3,12 @@ language: python
 python:
   - "3.5"
   - "3.6"
+  - "3.7-dev"
 install:
-  - pip install tox
+  - pip install -U setuptools
+  - pip install -U pip
+  - pip install -U wheel
+  - pip install -U tox
 script:
   - export TOXENV=py`python -c 'import sys; print("".join(map(str, sys.version_info[:2])))'`
```

Comment:
I suggest you to use an approach, I'm demoing in multidict PR now.

Reply:
What about https://github.com/tox-dev/tox-travis ?

Reply:
have `envlist = python` in `tox.ini` and run just `tox` then.